src/Symfony/Component/Notifier/Bridge/Sweego/composer.json

```
@@ -0,0 +1,36 @@
+{
+    "name": "symfony/sweego-notifier",
+    "type": "symfony-notifier-bridge",
+    "description": "Symfony Sweego Notifier Bridge",
+    "keywords": ["sms", "sweego", "notifier"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=8.2",
+        "symfony/http-client": "^6.4|^7.0",
+        "symfony/notifier": "^6.4|^7.0"
```

```suggestion
        "symfony/notifier": "^7.2"
```
Otherwise this bridge is not part of the `UnsupportedSchemeException`